### DISCUSSION TITLE
JSX title compiling Error - NextJS

### DISCUSSION THREAD

**Discussion by SH20RAJ** [vercel/next.js#63720] [Help]
### Link to the code that reproduces this issue

https://github.com/SH20RAJ/devart/commit/abde9d58651637636d9e81cae194d1f32df8fabf

### To Reproduce

```html
      <title>{ "DevArt - Programming Related Articles  - "+ params.slug}</title>
```
this is working but
```html
      <title>DevArt - Programming Related Articles  -  {params.slug}</title>
```
this is not 

any explanation ....

### Current vs. Expected behavior

```html
      <title>{ "DevArt - Programming Related Articles  - "+ params.slug}</title>
```
this is working but
```html
      <title>DevArt - Programming Related Articles  -  {params.slug}</title>
```
this is not 

it's returning just       `<title></title>`

any explanation ....

### Provide environment information

```bash
https://github.com/SH20RAJ/devart/commit/abde9d58651637636d9e81cae194d1f32df8fabf
```

### Which area(s) are affected? (Select all that apply)

App Router

### Which stage(s) are affected? (Select all that apply)

next dev (local), Vercel (Deployed)

### Additional context

```html
      <title>{ "DevArt - Programming Related Articles  - "+ params.slug}</title>
```
this is working but
```html
      <title>DevArt - Programming Related Articles  -  {params.slug}</title>
```
this is not 

any explanation ....

**Comment by devjiwonchoi** [comment]
This is how JSX works; take a look at https://github.com/vercel/next.js/discussions/38256#discussioncomment-3070196; it is well explained here.
